integration_tests/mockApis/elite2.ts

```diff
@@ -645,6 +647,34 @@ const stubGetOffenderDetails = ({
   return Promise.all([stubBasicInfo(), stubSentenceDetail(), stubSentenceTerms(), stubMainOffence()])
 }
 
+const stubGetOffenderDetailsBasic = ({
```

Comment:
Could you call stubGetOffenderDetails here with basic parameter set to true? The json body here overlaps with that functions, if basic is true should it be this smaller payload?

Reply:
implemented as suggested